backend/src/test/groovy/com/capstone/moneytree/controller/AlpacaControllerIT.groovy

```diff
@@ -58,6 +59,7 @@ class AlpacaControllerIT extends Specification {
         when: "Getting an Alpaca account"
         ResponseEntity<Account> response = alpacaController.getAccount(user.getId().toString())
 
+
```

Comment:
Your IDE is so weird for doing all these intends. I don't understand :D

Reply:
I think my IDE is using a function like: 
void messFormatUp(Format goodFormatting){
    messItUp(goodFromatting);
 }